frontend/public/components/overview/pods-overview.tsx

```diff
@@ -151,4 +149,6 @@ type PodOverviewListProps = {
 type PodsOverviewProps = {
   pods: PodKind[];
   obj: K8sResourceKind;
+  link?: string;
+  text?: string;
```

Comment:
These two fields should be more descriptive. Something like `allPodsLink` and `emptyText`.